File: CRS-GROUP-A-Angular-UI-UX-App/src/app/models/course-enrollment.ts
export class CourseEnrollment {

    courseId:number;
    courseName:string;
    studentId:number;
    studentName:string;
      
	constructor(courseId:number, courseName:string, studentId:number, studentName:string) {

		this.courseId = courseId;
		this.courseName = courseName;
		this.studentId = studentId;
		this.studentName = studentName;
	}
	
	getCourseName():string {
		return this.courseName;
	}

	setCourseName(courseName:string) {
		this.courseName = courseName;
	}

	getCourseId():number {
		return this.courseId;
	}

	setCourseId(courseId:number) {
		this.courseId = courseId;
	}

    getStudentId():number {
		return this.studentId;
	}

	setStudentId(studentId:number) {
		this.studentId = studentId;
	}

	getStudentName():string {
		return this.studentName;
	}

	setStudentName(studentName:string) {
		this.studentName = studentName;
	}
}

File: CRS-Group-A-Professor-Producer/src/main/java/com/lti/dto/CourseEnrollment.java
package com.lti.dto;

import java.io.Serializable;

import org.springframework.stereotype.Component;

@Component
public class CourseEnrollment implements Serializable {
	
    private int courseId;
    private String courseName;
    private int studentId;
    private String studentName;
    
    public CourseEnrollment() {}
      
	public CourseEnrollment(int courseId, String courseName, int studentId, String studentName) {

		this.courseId = courseId;
		this.courseName = courseName;
		this.studentId = studentId;
		this.studentName = studentName;
	}
	
	
	/**
	 * @return the courseName
	 */
	public String getCourseName() {
		return courseName;
	}

	/**
	 * @param courseName the courseName to set
	 */
	public void setCourseName(String courseName) {
		this.courseName = courseName;
	}

	/**
	 * @return the courseId
	 */
	public int getCourseId() {
		return courseId;
	}
	/**
	 * @param courseId the courseId to set
	 */
	public void setCourseId(int courseId) {
		this.courseId = courseId;
	}
	/**
	 * @return the studentId
	 */
	public int getStudentId() {
		return studentId;
	}
	/**
	 * @param studentId the studentId to set
	 */
	public void setStudentId(int studentId) {
		this.studentId = studentId;
	}
	/**
	 * @return the studentName
	 */
	public String getStudentName() {
		return studentName;
	}
	/**
	 * @param studentName the studentName to set
	 */
	public void setStudentName(String studentName) {
		this.studentName = studentName;
	}
    
   
}
